draft-wussler-openpgp-pqc.md

```diff
@@ -1057,47 +979,48 @@ scheme is as follows:
 
 ## Packet specifications
 
-### Public-Key Encrypted Session Key Packets (Tag 1) {#ecc-kyber-pkesk}
+### Public-Key Encrypted Session Key Packets (Tag 1) {#ecc-mlkem-pkesk}
 
 The algorithm-specific fields consists of:
 
  - A fixed-length octet string representing an ECC ephemeral public key in the
    format associated with the curve as specified in {{ecc-kem}}.
 
- - A fixed-length octet string of the Kyber ciphertext, whose length depends on
-   the algorithm ID as specified in {{tab-kyber-artifacts}}.
+ - A fixed-length octet string of the ML-KEM ciphertext, whose length depends
+   on the algorithm ID as specified in {{tab-mlkem-artifacts}}.
 
- - The one-octet algorithm identifier, if it was passed (in the case of a v3 PKESK packet).
+ - The one-octet algorithm identifier, if it is passed (in the case of a v3
+   PKESK packet).
 
  - A variable-length field containing the wrapped session key:
 
    - A one-octet size of the following field;
 
-   - The wrapped session key represented as an octet string, i.e.,
-     the output of the encryption procedure described in
-     {{ecc-kyber-encryption}}.
+   - The wrapped session key represented as an octet string, i.e., the output
+     of the encryption procedure described in {{ecc-mlkem-encryption}}.
 
-Note that unlike most public-key algorithms, in the case of a v3 PKESK packet, the symmetric algorithm identifier is not encrypted.
-Instead, it is prepended to the encrypted session key in plaintext.
-In this case, the symmetric algorithm used MUST be AES-128, AES-192 or AES-256 (algorithm ID 7, 8 or 9).
+Note that unlike most public-key algorithms, in the case of a v3 PKESK packet,
+the symmetric algorithm identifier is not encrypted.  Instead, it is prepended
+to the encrypted session key in plaintext.  In this case, the symmetric
+algorithm used MUST be AES-128, AES-192 or AES-256 (algorithm ID 7, 8 or 9).
```

Comment:
```suggestion
algorithm used MUST be AES-128, AES-192 or AES-256.
```
Is it really necessary to list the algorithm IDs for AES from another spec?

Reply:
I don't think it's necessary, but makes the dev life slightly easier IMO

Reply:
So keep it as it was (listing the IDs)?
It's ok for me.

Reply:
Yes, as much as we list the IDs in the hash binding tables

Reply:
Ok, then lets drop this suggestion.